cves/kernel/CVE-2019-12378.yml

```diff
@@ -169,8 +169,9 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: |
+    Yes, the fix is to check the return value and handle an error. An automated tool can find this.
```

Comment:
Could you explain more about how it would find it?  What tool?

Reply:
updated